authors/apps/authentication/models.py

```diff
@@ -1,18 +1,18 @@
 import jwt
-
```

Comment:
Please leave this empty line alone.

Reply:
I was trying to group library imports together